config/rbac/instascale_role.yaml

```diff
@@ -0,0 +1,102 @@
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  creationTimestamp: null
+  name: instascale-role
+rules:
+- apiGroups:
+  - ""
+  resources:
+  - configmaps
+  verbs:
+  - get
+- apiGroups:
+  - ""
```

Comment:
It's already part of the "core" CusterRole, so it's preferable not duplicating it.